src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -157,6 +158,21 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
  */
 - (void)shutdown;
 
+/**
+ * Discover commissionable nodes.
+ */
+- (void)discoverCommissionableNodes;
```

Comment:
Why is this API desirable?  Especially since it does not actually do what the name says, in that it does only DNS-SD discovery.

Reply:
I removed all discovery and ethernet pairing changes because they are useful for test but not a mandatory feature for Matter certification.